src/pages/_nav_bar.tsx

```diff
@@ -74,7 +75,10 @@ function NavBarComponent({ customMouseEnter, customMouseLeave }: any) {
                   onMouseEnter={customMouseEnter}
                   onMouseLeave={customMouseLeave}
                 >
-                  {menu_text}
+                <div>
+                  <Link to={`/${menu_text}/`}>{menu_text}</Link>
```

Comment:
The app doesn't use routing for the pages, it uses a single view portal and manage child components to be displayed based on state value.